user-management-service/app/build.gradle

```diff
@@ -0,0 +1,37 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.1.4.RELEASE")
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+
+bootJar {
+    baseName = 'usermanagement-app'
+    version = applicationVersion
+    enabled = true
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile("org.springframework.boot:spring-boot-starter-web")
```

Comment:
I did say use laundry service gradle files as example. This file formatting is bad

Reply:
fixed

Reply:
I see that the dependencies have different formatting. We should use single style for this

Reply:
In laundry service the dependencies are formatted consistently!